runtime/oti/j9nonbuilder.h

```diff
@@ -1398,6 +1398,7 @@ typedef struct J9RAS {
 	UDATA pid;
 	UDATA tid;
 	char* serviceLevel;
+	char* productName;
```

Comment:
The type should be `const char *` to reflect the fact that the string should not be modified nor freed via this reference:
```c
	const char *productName;
```